AUTHORS.md

```diff
@@ -1 +1,17 @@
-FIXME: list authors' names and email addresses.
\ No newline at end of file
+Thank you to the following contributors (in alphabetical order by user name):
+* Ngumih Fien (@fenne)
+* Bruno P. Kinoshita (@kinow) https://orcid.org/0000-0001-8250-4074
+* Peter Amstutz (@tetron) https://orcid.org/0000-0003-3566-7705
+* Toby Hodges (@tobyhodges) https://orcid.org/0000-0003-1766-456X
+* Michele Mastropietro (@elehcim) https://orcid.org/0000-0002-6324-5713
+* Robin Long (@longr) 
+* Manabu Ishii (@manabuishii)
+* Michael Crusoe (@mr-c) http://orcid.org/0000-0002-2961-9670
+* @rmccole
+* Adrian Sharma (@sharmatime)
+* Oghenemaro Akpobi (@Smyja)
+* Stian Soiland-Reyes (@stain) [https://orcid.org/0000-0001-9842-9718
+   
+Thanks to:
```

Comment:
```suggestion
* Benjamin Carr (@BenjaminHCCarr) <https://orcid.org/0000-0002-6206-4638>
* Emannuel Fernandes de Oliveira Carvalho (@emannuelOC)
* Eugene Miloslavsky (@emiloslavsky)
* Henry Liu (@HenryLiu0)
* Ngumih Fien (@Fienne)
* @jessMaia
* Bruno P. Kinoshita (@kinow) <https://orcid.org/0000-0001-8250-4074>
* Konstantin Taletskiy (@ktaletsk) <https://orcid.org/0000-0001-6846-9815>
* Kushal Beniwal (@KushalBeniwal)
* Peter Amstutz (@tetron) <https://orcid.org/0000-0003-3566-7705>
* Toby Hodges (@tobyhodges) <https://orcid.org/0000-0003-1766-456X>
* Michele Mastropietro (@elehcim) <https://orcid.org/0000-0002-6324-5713>
* Robin Long (@longr) <https://orcid.org/0000-0003-2249-645X>
* Mark A. Jensen  (@majensen) <https://orcid.org/0000-0001-5215-101X>
* Manabu Ishii (@manabuishii) <https://orcid.org/0000-0002-5843-4712>
*  @medcelerate
* Michael Crusoe (@mr-c) <http://orcid.org/0000-0002-2961-9670>
* Ruth B. McCole  (@rmccole) <https://orcid.org/0000-0002-1861-7055>
* @sersorrel
* @swzCuroverse
* Adrian Sharma (@sharmatime)
* Oghenemaro Akpobi (@Smyja)
* Stian Soiland-Reyes (@stain) <https://orcid.org/0000-0001-9842-9718>
*  Daiki Tsuchiya (@tsuchi8) <https://orcid.org/0000-0003-2112-9316>
* Tomoya Tanjo (@tom-tan) <https://orcid.org/0000-0002-4421-9659>
* Wolfgang Gerlach (@wgerlach) <https://orcid.org/0000-0002-1480-1242>
* Zipho Mashologu (@zipho)

Thanks to:
```
rmccole's name is from her repo <https://github.com/rmccole/UCEs_genome_organization>

Reply:
I can't review this cause it is against my own pull request for some reason.